great_expectations/validator/metrics_calculator.py

```diff
@@ -46,9 +46,12 @@ def columns(self, domain_kwargs: Optional[Dict[str, Any]] = None) -> List[str]:
         Convenience method to run "table.columns" metric.
         """
         if domain_kwargs is None:
-            domain_kwargs = {
```

Comment:
This change is consistent with the way "active `Batch`" works -- and is a cleaner coding style.